.prettierrc.json

```diff
@@ -1,20 +1,6 @@
 {
     "$schema": "http://json.schemastore.org/prettierrc",
-
-    "parser": "typescript",
-    "printWidth": 120,
-    "semi": true,
     "singleQuote": true,
-    "tabWidth": 4,
-    "trailingComma": "es5",
-
-    "overrides": [
-        {
-            "files": ["*.md"],
-            "options": {
-                "tabWidth": 2
```

Comment:
Why for `.md` we use 2 spaces and not 4?

Reply:
Probably a left-over from the original template. Seems fine to me to just be the same as everything else.